### DISCUSSION TITLE
xgterm installation on fedora 39

### DISCUSSION THREAD

**Discussion by ShahanaSaifi** [iraf-community/iraf#386] [Q&A] [answered]
how to install xgterm on fedora ?
I have install iraf and ds9 but the graphs are not working because of xgterm!
after giving $ sudu dnf install xgterm , I am getting  "error: unable to find a match:xgterm" .

**Comment by olebole** [comment, marked as answer]
I moved this to the discussion because it is not a bug of **x11iraf**.

As far as I know there is no **x11iraf** (or **xgterm**) package available for Fedora Linux, so you probably need to compile it yourself. This is not complicated; see the [x11iraf web page](https://iraf-community.github.io/x11iraf.html).

You can contribute to Fedora (and the IRAF community) by packaging **x11iraf** if you have some spare time.
